source/index.js

```diff
@@ -177,6 +180,7 @@ module.exports = async (input = 'patch', options) => {
 		tasks.add([
 			{
 				title: `Publishing package using ${pkgManagerName}`,
+				skip: () => options.preview,
```

Comment:
I'm not sure if the best way to implement this would be to simply skip the task.
I think this flag should be somewhat of a `--verbsose` flag as well, meaning that it should print out the _exact_ commands it's going to run (e.g. `npm version patch`), as well as what made it decide to run those specific commands.

Reply:
The commands could be printed in skip:
![image](https://user-images.githubusercontent.com/5731176/57185435-d1ca5900-6ea1-11e9-9dbb-e0e53c271539.png)

Reply:
Despite me approving this PR, I still feel like the current way preview mode deals with this could be improved.